arranger/configmaps.yml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: arranger-server-config
+data:
+  ES_HOST: http://elasticsearch-workers:9200/
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: arranger-admin-ui-config
+data:
+  HOST: http://arranger-server.qa.clin.ferlab.bio
```

Comment:
This url is very environment-specific. I'd leave it with the value "__UNDEFINED__" and overwrite it with a mergepatch in cqdg-environments.